### ISSUE TITLE
Ctrl-l panics on Linux

### ISSUE THREAD

**Issue by jc00ke** [jwilm/alacritty#162]
On 852c2d8 I get the following backtrace

```
> env RUST_BACKTRACE=1 alacritty
# I hit Ctrl+l (that's an L)
thread 'pty reader' panicked at 'index out of bounds: the len is 24 but the index is 18446744073709551615', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcollections/vec.rs:1371
stack backtrace:
   1:     0x55ad85cbd2ba - std::sys::imp::backtrace::tracing::imp::write::h2b860f17b6b0f511
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:42
   2:     0x55ad85cc262f - std::panicking::default_hook::{{closure}}::hedea7d569eb1e6b8
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:349
   3:     0x55ad85cc222e - std::panicking::default_hook::he87e0c76c89c52a4
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:365
   4:     0x55ad85cc2ad7 - std::panicking::rust_panic_with_hook::h304127bde2d3d5e2
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:553
   5:     0x55ad85cc2964 - std::panicking::begin_panic::h814f60faa3ce8314
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:515
   6:     0x55ad85cc2889 - std::panicking::begin_panic_fmt::hf4f7d700fa077693
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:499
   7:     0x55ad85cc2817 - rust_begin_unwind
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:475
   8:     0x55ad85cee8dd - core::panicking::panic_fmt::hee207ff21fb62166
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/panicking.rs:69
   9:     0x55ad85cee888 - core::panicking::panic_bounds_check::hca28eae069525a56
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcore/panicking.rs:56
  10:     0x55ad85bb809e - <alacritty::ansi::Performer<'a, H, W> as vte::Perform>::print::hefa9b8d1fdb41d3b
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libcollections/vec.rs:1371
                        at /home/jesse/src/alacritty/src/grid.rs:310
                        at /home/jesse/src/alacritty/src/grid.rs:224
                        at /home/jesse/src/alacritty/src/term/mod.rs:621
                        at /home/jesse/src/alacritty/src/ansi.rs:438
  11:     0x55ad85bb0b5d - <alacritty::event_loop::EventLoop<Io>>::spawn::{{closure}}::hedf7c076eee628c6
                        at /home/jesse/.cargo/registry/src/github.com-1ecc6299db9ec823/vte-0.1.2/src/lib.rs:167
                        at /home/jesse/.cargo/registry/src/github.com-1ecc6299db9ec823/vte-0.1.2/src/lib.rs:137
                        at /home/jesse/src/alacritty/src/ansi.rs:85
                        at /home/jesse/src/alacritty/src/event_loop.rs:211
                        at /home/jesse/src/alacritty/src/event_loop.rs:308
  12:     0x55ad85b9c1db - std::panicking::try::do_call::h0f7314b05607066d
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panic.rs:295
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:458
  13:     0x55ad85cc9a1a - __rust_maybe_catch_panic
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libpanic_unwind/lib.rs:98
  14:     0x55ad85bae9ca - <F as alloc::boxed::FnBox<A>>::call_box::h4b6067876e9dfe43
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panicking.rs:434
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/panic.rs:351
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/thread/mod.rs:287
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/liballoc/boxed.rs:605
  15:     0x55ad85cc1924 - std::sys::imp::thread::Thread::new::thread_start::h460a0adc61139b06
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/liballoc/boxed.rs:615
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/sys_common/thread.rs:21
                        at /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/libstd/sys/unix/thread.rs:84
  16:     0x7f5cb4bfb6c9 - start_thread
  17:     0x7f5cb471e0ae - __clone
  18:                0x0 - <unknown>
Goodbye
```

It doesn't crash when I hit `Ctrl+l` in `tmux`.

This project is fantastic! Great work! :tada: :fireworks: :balloon:

**Comment by CarwynNelson** [comment]
Just to add to this. ctrl+l does not cause a crash on MacOS so it seems this bug is specific to linux.

**Comment by deferraz** [comment]
I've built a release today on Debian Unstable and could use ctrl+l without problems

**Comment by jwilm** [comment]
May have been fixed in #216 #156 #164. Can you please try it out?

**Comment by jc00ke** [comment]
Resolved, thanks!
